docs/tutorial/tutorial-0.rst

```diff
@@ -14,10 +14,16 @@ The first thing we'll need is a working Python interpreter.
 
   .. group-tab:: macOS
 
-    If you're on macOS, you can get the official installer from `the Python
-    website <https://www.python.org/downloads>`_. You can use any stable version
-    of Python from 3.8 onward. We'd advise avoiding alphas, betas, and release
-    candidates unless you *really* know what you're doing.
+    If you're on macOS, a recent version of Python is included with Xcode or
+    the command line developer tools. To check if you already have it, run the
+    following command:
```

Comment:
Hrm - while this is true, I'm not sure how I feel about it. Apple has a notorious history of... "fixing"... Python; and not keeping up to date. My gut tells me it's better to be explicit here, and ask users to install an official Python distribution, even though *technically* one is available in the Command Line tools.

Reply:
Following up after a conversation in person; since you can complete the tutorial with the command line tools version of Python, there's a comment further down about using Python from other sources, and you're going to have the command line tools anyway, we might as well go with it for now. If it turns out to be a problem in future (or they don't update from Python3.9 in 2 years time), we can revisit this.